skyportal/handlers/api/spectrum.py

```diff
@@ -38,6 +38,11 @@ def post(self):
         """
         data = self.get_json()
         instrument_id = data.get('instrument_id')
+        if isinstance(instrument_id, list):
```

Comment:
Wait, I thought this should always be a single inst ID??

Reply:
It should; this is just a safe-guard in case someone provides many.

We could take it out, but I imagined sometimes people would split a spreadsheet with "instrument_id" as a field (i.e., in every row), and I don't want to prevent it in case this is valid input.

But, I don't feel strongly about it either way.

Reply:
Oh, I see, a homogeneous list.. yeah this seems fine.